docs/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 ## Catalog, Lambdas
 !-->
+
+## Python API
+* [Changed] Switch from a regular SHA256 checksum to a hash list to match S3's built-in checksums ([#2782](https://github.com/quiltdata/quilt/pull/2782))
+* [Changed] Delay object hashing until package push to take advantage of S3's hashing ([2782](https://github.com/quiltdata/quilt/pull/2782))
```

Comment:
I think breaking changes should be listed more explicitly.

Reply:
@akarve: what should we say here?

Reply:
Well, I think that `top_hash` is not passed to `dest` function anymore.